code/zoltar_scripts/upload_covid19_forecasts_to_zoltar.py

```diff
@@ -187,8 +216,8 @@ def upload_covid_all_forecasts(path_to_processed_model_forecasts, dir_name):
                 else:
                     try:
                         logger.debug('Upload forecast for model: %s \t|\t File: %s\n' % (metadata['model_abbr'],forecast))
-                        util.upload_forecast(conn, quantile_json, forecast,
-                                             project_name, metadata['model_abbr'], time_zero_date,
+                        upload_covid_forecast_by_model(conn, quantile_json, forecast,
+                                             project_name, model, metadata['model_abbr'], time_zero_date,
```

Comment:
The default for `sync` is `True`. do we want that so? I mean, in this way, won't it be the same (busy polling for every model upload) as before?

Reply:
Having `sync=True` I think is needed because it calls this busy_poll_job() which gives us the status of the upload, and if it fail we can eventually log the message of why it fail. What is different from before with my change is removing the polling for list of projects and models at this [line ](https://github.com/reichlab/zoltpy/blob/master/zoltpy/util.py#L133) of util.upload_forecast() in zoltpy

Reply:
Got it! Looks good 👍